src/app/ngrx/reducers/podcast-ranks.reducer.spec.ts

```diff
@@ -19,12 +19,12 @@ describe('Podcast Ranks Reducer', () => {
   });
 
   it('should set loading, loaded, and unset error on podcast ranks load', () => {
-    const { group, interval, beginDate, endDate } = routerParams;
+    const { podcastId, group, interval, beginDate, endDate } = routerParams;
     const newState = reducer(initialState,
-      new ACTIONS.CastlePodcastRanksLoadAction({id: routerParams.podcastId, group, interval, beginDate, endDate}));
-    expect(newState.loading).toBeTruthy();
-    expect(newState.loaded).toBeFalsy();
-    expect(newState.error).toBeNull();
+      new ACTIONS.CastlePodcastRanksLoadAction({id: podcastId, group, interval, beginDate, endDate}));
```

Comment:
I put loading, loaded, and error on the individual entities in order to support the loading and error status for the nested table.